tests/e2e/e2e_test.go

```diff
@@ -44,11 +44,11 @@ func TestOdo(t *testing.T) {
 	RunSpecs(t, "odo test suite")
```

Comment:
Remove the suite function in a separate file to isolate it form the actual test

For example: 

Create a file ```odo_suite_test.go```
```
package e2e

import (
	"testing"

	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
)

func TestOdo(t *testing.T) {
	RegisterFailHandler(Fail)
	RunSpecs(t, "odo test suite")
}
```

Reply:
Agree, but I think that this goes a little bit beyond creating an example of how to write tests.
This should be part of the cleaning up effort .

Reply:
This PR is intended to provide just a clean direction and template how we can structure our tests.
This should be part of a cleanup effort in a separate PR.